### DISCUSSION TITLE
Converting question labeling issue to Discussions

### DISCUSSION THREAD

**Discussion by tkoyama010** [pyvista/pyvista#1397] [Ideas]
We can now convert [question labeled](https://github.com/pyvista/pyvista/issues?q=is%3Aissue+label%3Aquestion+) issues to Discussion as a issue triage and organize of Q&A. How do you think about it?

![image](https://user-images.githubusercontent.com/7513610/122699473-74154880-d284-11eb-9628-616998f78aa4.png)

**Comment by akaszynski** [comment]
Agreed.  And if using discussions works out, we can also transfer support issues to pyvista and then transfer them to discussions.
